std/format.d

```diff
@@ -2666,6 +2672,37 @@ if (is(FloatingPointTypeOf!T) && !is(T == enum) && !hasToString!(T, Char))
     }
 }
 
+///
+@safe unittest
+{
+    import std.array : appender;
+    auto w = appender!string();
+
+    auto spec = singleSpec("%.1f");
+    formatValue(w, 1337.7, spec);
+    assert(w.data == "1337.7");
+
+    w = new appender!string();
+    spec = singleSpec("%,3.2f");
+    formatValue(w, 1331.982, spec);
+    assert(w.data == "1,331.98");
```

Comment:
It might be easier to use `assert(format("%...", 123) == "...")` for the rest of the examples?

Reply:
No problem, thank you for the hint.